website/src/components/Buttons/Skip.tsx

```diff
@@ -11,13 +11,15 @@ import {
   Textarea,
   useDisclosure,
 } from "@chakra-ui/react";
+import { useTranslation } from "next-i18next";
 import { useState } from "react";
 
 interface SkipButtonProps extends ButtonProps {
   onSkip: (reason: string) => void;
 }
 
 export const SkipButton = ({ onSkip, ...props }: SkipButtonProps) => {
+  const { t } = useTranslation("common");
```

Comment:
I'm not sure, but maybe you don't need to specify the namespace when the only one used is `common`?

Reply:
You are right. I've changed it in all the files that only used `common`.